src/scripts/main.js

```diff
@@ -2,7 +2,30 @@
 
 const people = require('./lib/people');
 
-// eslint-disable-next-line no-console
-console.log(people); // you can remove it
+function createTable(peoplesArr) {
+  const table = document.querySelector('tbody');
 
-// write your code here
+  for (const person of people) {
+    const tableRow = document.createElement('tr');
+
+    const personData = {
+      name: person.name,
+      gender: person.sex === 'm' ? 'Male' : 'Female',
+      born: person.born,
+      died: person.died,
+      Age: (person.died - person.born),
+      Century: (Math.ceil(person.died / 100)),
```

Comment:
Don't use capital letters for object properties
```suggestion
      age: (person.died - person.born),
      century: (Math.ceil(person.died / 100)),
```